lasy/utils/denoise.py

```diff
@@ -0,0 +1,103 @@
+from lasy.profiles.transverse.hermite_gaussian_profile import (
+    HermiteGaussianTransverseProfile,
+)
+from lasy.profiles.transverse.transverse_profile import TransverseProfile
+from lasy.utils.mode_decomposition import hermite_gauss_decomposition
+
+
+def denoise_transverse_hg(
+    transverse_profile,
+    wavelength,
+    resolution=0.2e-6,
+    lo=[-2e-4, -2e-4],
+    hi=[2e-4, 2e-4],
+    n_modes_x=2,
+    n_modes_y=2,
```

Comment:
I think only `n_modes_x/y` should be optional, and given a meaningful value, e.g., 10. For the others, the user will get something likely wrong and confusing with default values.